pkgs/development/compilers/bluespec-bsc/default.nix

```diff
@@ -0,0 +1,104 @@
+{ stdenv
+, fetchFromGitHub
+, fetchpatch
+, autoconf
+, automake
+, bison
+, flex
+, fontconfig
+, ghc
+, glibc
+, gmp
+, gperf
+, haskell
+, libX11
+, libpoly
+, perl
+, pkgconfig
+, verilog
+, xorg
+, zlib
+}:
+
+let
+  gmp-static = gmp.override { withStatic = true; };
+  ghcWithPackages = haskell.packages.ghc844.ghc.withPackages (g: (with g; [old-time regex-compat syb]));
```

Comment:
We can just use the default Haskell compiler, since Bluespec should compile with any choice, at the moment.

Reply:
Unfortunately, it doesn't. I run into 

```
/build/source/inst/bin/bsc -stdlib-names -bdir /build/source/build/bsvlib -p . -vsearch /build/source/build/bsvlib -no-use-prelude PreludeBSV.bsv
imperative statement (not BVI context)make[3]: *** [Makefile:25: /build/source/build/bsvlib/PreludeBSV.bo] Error 1
```

when trying to compile it with our default ghc (8.6.5 currently)

I added a comment in https://github.com/B-Lang-org/bsc/issues/20#issuecomment-583724030.

Reply:
ugh, thanks for the heads up. Annoying.